service/service.go

```diff
@@ -56,10 +56,17 @@ var getObservationStore = func(ctx context.Context) (observationStore *graph.DB,
 	return observationStore, errorConsumer, nil
 }
 
+var pConfig *kafka.ProducerConfig
```

Comment:
I think this variable is not needed, you already define `pConfig` inside getProducer (I think it's the right place)